docs/source/before_start/architecture.rst

```diff
@@ -87,7 +87,7 @@ Distinctive features of YAC algorithm are its scalability, performance and Crash
 To ensure consistency in the network, if there are missing blocks, they will be downloaded from another peer via `Synchronizer <#synchronizer>`__.
 Committed blocks are stored in `Ametsuchi <#ametsuchi>`__ block storage.
 
-For general definition of the consensus, please check `this link <../core_concepts/glossary.html#consensus>`_.
+For general definition of the consensus, please check `this link <glossary.html#consensus>`_.
 
 
 Synchronizer
```

Comment:
Broken link on line 97, should be `peers' <glossary.html#peer>`

Reply:
Got it. Sublime does not highlight some links for some reason.